src/testIntegration/java/uk/gov/justice/digital/delius/controller/api/ContactAPITest.java

```diff
@@ -42,6 +42,7 @@ public void setup() {
     }
 
     @Test
+    // TODO DT-835 This test intermittently fails due to data being corrupted by another test - currently working in Circle by splitting the tests into parallel executions, but needs a proper fix
```

Comment:
I think you can remove this comment now

Reply:
I've just added it :)  Just wanted to put something in there in case somebody else notices the failing test and thinks they did bad.

Reply:
OK